src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/RazorLanguageServer.cs

```diff
@@ -71,8 +71,8 @@ public static Task<RazorLanguageServer> CreateAsync(Stream input, Stream output,
                     .WithInput(input)
                     .WithOutput(output)
                     .ConfigureLogging(builder => builder
-                        .AddLanguageServer()
-                        .SetMinimumLevel(RazorLSPOptions.GetLogLevelForTrace(trace)))
+                        .AddLanguageServer(RazorLSPOptions.GetLogLevelForTrace(trace))
+                        .SetMinimumLevel(LogLevel.Trace)) // We set the minimum level here to "Trace" to ensure that other providers still get the opportunity to act on logs if they prefer.
```

Comment:
Will this work?

From below: https://github.com/OmniSharp/csharp-language-server-protocol/issues/106

Reply:
It works in practice 😄. Something right must be happening